psydac/cad/geometry.py

```diff
@@ -446,7 +446,7 @@ def refine_nurbs(nrb, ncells=None, degree=None, multiplicity=None, tol=1e-9):
         multiplicity of each knot in the knot sequence in each direction
 
     tol : <float>
-        tolerance
```

Comment:
Don't you mean this instead?
```python
"""
tol : <float>
    Minimum distance between two break points.
"""
```
To which I would add 

"Two knots with a distance less than `tol` are considered to be repeated knots, which correspond to the same break points"

Reply:
done